ui/public/locales/en/translation.json

```
@@ -867,15 +867,19 @@
         "delete": {
           "description": "Are you sure you want to delete <b>{{name}}</b>?<br/> This cannot be undone."
         },
+        "form": {
+          "unsupported": "This mime type is not supported",
```

```suggestion
          "unsupported": "Unsupported MimeType: Only text-based files can be used for secrets",
```
I found this message slightly confusing when testing with a PNG so I added some clarification.